tests/snapshots/src/main/generated/com/airbnb/epoxy/EpoxyAsyncUtil.java

```diff
@@ -148,6 +149,7 @@ public static Handler createHandler(Looper looper, boolean async) {
     }
 
     return new Handler(looper);
+//         ^^^^^^^^^^^^^^^^^^^ reference `<init>`#
```

Comment:
This looks like an improvement 👍

Reply:
Im assuming this isnt correct yet, but progress?